radio/src/targets/flysky/adc_driver.cpp

```diff
@@ -100,9 +114,14 @@ void adcInit()
   ADC_Init(ADC1, &adc_init);
 
   // configure each channel
+  #if defined(FLYSKY_GIMBAL)
```

Comment:
Use !defined and keep this line same for both conditions to simplify it

Reply:
Thanks, done!